lib/core/test/manual/Cardano/Launcher/POSIXSpec.md

```diff
@@ -32,14 +32,14 @@ signals are correctly handled on a unix system.
 
 ### `SIGQUIT`
 
-- Start the launcher using `stack exec -- cardano-wallet launch`
+- Start the launcher using `stack exec -- cardano-wallet-jormungandr launch --genesis-block lib/jormungandr/test/data/jormungandr/block0.bin --bft-leaders lib/jormungandr/test/data/jormungandr/secret.yaml`
 - In another terminal, run `ps -ef | grep cardano`, there should be three
   processes running (the launcher, the wallet and the underlying chain
   producer)
   ```
   $ ps -ef | grep cardano
   10983  4904  0 09:42 pts/1 00:00:00 /home/user/Documents/IOHK/cardano-wallet/.stack-work/install/x86_64-linux/lts-13.8/8.6.3/bin/cardano-wallet
-  11003 10983 75 09:42 pts/1 00:03:30 cardano-http-bridge start --port 8080
+  11003 10983 75 09:42 pts/1 00:03:30 wallet-jormungandr start --port 8080
```

Comment:
This is wrong. `cardano-http-bridge` referred to the actual bridge application. Here we should be referring to `jormungandr` instead.